app/models/course-language-configuration.ts

```diff
@@ -0,0 +1,27 @@
+import Model from '@ember-data/model';
+import { attr, belongsTo } from '@ember-data/model';
+import { equal } from '@ember/object/computed'; // eslint-disable-line ember/no-computed-properties-in-native-classes
+
+import type CourseModel from 'codecrafters-frontend/models/course';
+import type LanguageModel from 'codecrafters-frontend/models/language';
+import type UserModel from 'codecrafters-frontend/models/user';
+
+export default class CourseLanguageConfigurationModel extends Model {
+  @attr() declare alphaTesterUsernames: Array<string>;
+  @attr('string') declare releaseStatus: string;
```

Comment:
```suggestion
  @attr('string') declare releaseStatus: 'a' | 'b' | 'c';
```

(Check core for exact values)